README.md

```diff
@@ -1,43 +1,56 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# ezCar frontend
-
-> This project consumes our own API to allow users to rent a car in a React UI.
+# ezCar Frontend
 
+> This application allows users to book a car present in our database. The user can set the reservation date and final date for a selected car. If the user is an administrator they can upload a car and delete it from our database.
```

Comment:
- [ ] Please, add a screenshot to showcase your frontend project 💪 

(This review comment was originally mistakenly posted on the backend PR.)

Reply:
@Meltrust Thank you for your recommendation. We implemented the change as requested